modules/private-clusters-setting-dns-private.adoc

```diff
@@ -42,22 +49,24 @@ status: {}
 +
 Note that the `spec` section contains both a private and a public zone.
 
-. Patch the `DNS` custom resource to remove the public zone:
+. Patch the `DNS` CR to remove the public zone:
 +
 [source,terminal]
 ----
 $ oc patch dnses.config.openshift.io/cluster --type=merge --patch='{"spec": {"publicZone": null}}'
 dns.config.openshift.io/cluster patched
```

Comment:
Definitely out of scope and not necessary for you to fix, but if you were interested - this line, which appears to be output, probably should be in its own `example output` block

Reply:
Great catch, fixed it since I could see this causing issues for users tbh! Thanks!